tools/utils/Utils/AppxPackaging/ChildPackageMetadata.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Msix.Utils.AppxPackaging
+{
+    using Microsoft.Msix.Utils.AppxPackagingInterop;
+
+    /// <summary>
+    /// The type of the package in a bundle.
+    /// </summary>
+    public enum PackageType
+    {
+        /// <summary>
+        /// A package containing executable code.
+        /// </summary>
+        Application = 0,
+
+        /// <summary>
+        /// A resource package.
+        /// </summary>
+        Resource = 1,
+    }
+
+    /// <summary>
+    /// Class that represents the information of a bundled child package.
+    /// </summary>
+    public class ChildPackageMetadata
+    {
+        public ChildPackageMetadata(string filePath, APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE packageType, ulong packageSize, string packageFullName, ulong packageVersion, string packageResourceId)
+        {
+            this.FilePath = filePath;
+            this.PackageType = (PackageType)packageType;
+            this.PackageSize = packageSize;
+            this.PackageFullName = packageFullName;
+            this.PackageArchitecture = PackagingUtils.GetPackageArchitectureFromFullName(packageFullName);
+            this.PackageVersion = new VersionInfo(packageVersion);
+            this.PackageResourceId = packageResourceId;
```

Comment:
Nit: I don't think all properties need to be prefixed with `Package`. I think it would be good to follow the pattern from existing types. Looking at the constructor call and `PackageMetadata` this would be `PackageType`, `Size`, `PackageFullName`, `Architecture`, `Version`, `ResourceId`.

Reply:
Done